.github/workflows/ci.yaml

```diff
@@ -359,7 +359,12 @@ jobs:
           path: ./.yarn/
           restore-keys: |
             ${{ runner.os }}-yarn-${{ hashFiles('./yarn.lock') }}
+      - name: Installing AreTheTypesWrong Library
+        run: npm i -g @arethetypeswrong/cli
```

Comment:
@ruzell22 Please declare dependencies in the package json file instead of manually installing them in a script.